nx_parallel/utils/tests/test_chunk.py

```diff
@@ -17,17 +16,19 @@ def test_get_n_jobs():
 
         # Test with n_jobs set to negative value
         assert nxp.get_n_jobs(-1) == os.cpu_count()
-        nx.config.backends.parallel.active = False
+
+        # Test with joblib's context
         from joblib import parallel_config
 
-        parallel_config(n_jobs=3)
-        assert nxp.get_n_jobs() == 3
-        nx.config.backends.parallel.active = True
-        nx.config.backends.parallel.n_jobs = 5
-        assert nxp.get_n_jobs() == 5
-        # restore nx.config
-        nx.config.backends.parallel.active = False
-        nx.config.backends.parallel.n_jobs = None
+        with parallel_config(n_jobs=3):
+            assert nxp.get_n_jobs() == 3
+
+        # Test with nx-parallel's context
+        nxp_config = nx.config.backends.parallel
+
+        with nxp_config(active=True, n_jobs=5):
```

Comment:
no need to create a variable

```suggestion
        with nx.config.backends.parallel(active=True, n_jobs=5):
```